develop/application-devs/interact/understand-parachain-data.md

```diff
@@ -0,0 +1,172 @@
+
+Metadata provides a complete inventory of a chain's runtime and is key to enabling client applications to interact with the node, parse responses, and correctly format message payloads sent back to that chain.
+
+## Generating Metadata
+
+To efficiently use the blockchain's networking resources and minimize the data transmitted over the network, the metadata schema is encoded using the [Parity SCALE Codec](https://github.com/paritytech/parity-scale-codec?tab=readme-ov-file#parity-scale-codec){target=\_blank}. This encoding is done automatically through the [`scale-info`](https://docs.rs/scale-info/latest/scale_info/){target=\_blank}crate.
+
+At a high level, generating the metadata involves the following steps:
+
+1. The pallets in the runtime logic expose callable functions, types, parameters, and documentation that need to be encoded in the metadata
+2. The `scale-info` crate collects type information for the pallets in the runtime, builds a registry of the pallets that exist in a particular runtime, and the relevant types for each pallet in the registry. The type information is detailed enough to enable encoding and decoding for every type
+3. The [`frame-metadata`](https://github.com/paritytech/frame-metadata){target=\_blank} crate describes the structure of the runtime based on the registry provided by the `scale-info` crate
+4. Nodes provide the RPC method `state_getMetadata` to return a complete description of all the types in the current runtime as a hex-encoded vector of SCALE-encoded bytes
+
+## Getting Metadata for a Runtime
+
+The type information provided by the metadata enables applications to communicate with nodes using different versions of the runtime and across chains that expose different calls, events, types, and storage items. The metadata also allows libraries to generate a substantial portion of the code needed to communicate with a given node, enabling libraries like [`subxt`](https://github.com/paritytech/subxt){target=\_blank} to generate front-end interfaces that are specific to a target chain.
+
+### Using Polkadot.js
+
+Visit the [Polkadot.js Portal](https://polkadot.js.org/apps/#/rpc){target=\_blank} and select the **Developer** dropdown in the top banner. Select **RPC Calls** to make the call to request metadata. Follow these steps to make the RPC call:
+
+1. Select **state** as the endpoint to call
+2. Select **`getMetadata(at)`** as the method to call
+3. Click **Submit RPC call** to submit the call and return the metadata in JSON format
+
+![Get metadata with Polkadot.js](/images/develop/application-devs/interact/understand-parachain-data/metadata-01.webp)
+
+### Using curl For Metadata Retrieval
```

Comment:
```suggestion
### Using Curl For Metadata Retrieval
```

They refer to it as Curl in the README of the source repo: https://github.com/curl/curl
So lets go with that